ofrak_core/ofrak/resource.py

```diff
@@ -934,25 +873,6 @@ def get_attributes(self, attributes_type: Type[RA]) -> RA:
         )
         return attributes
 
-    def get_all_attributes(self) -> Iterable[ResourceAttributes]:
-        """
-        Get values for all the attributes this resource has.
-        :return:
-        """
-        return list(self._resource.attributes.values())
-
-    def remove_attributes(self, attributes_type: Type[ResourceAttributes]):
```

Comment:
Let's keep this around, even if unused. It would be simple to test, and it is basic functionality that pairs with `add_attributes`.

Reply:
Ok, I've added it back with a test.